lib/active_triples/persistence_strategies/parent_strategy.rb

```diff
@@ -30,7 +30,21 @@ def persisted?
     end
 
     ##
-    # Destroys the resource by removing it graph and references from the 
+    # Mark that this resource has not yet been loaded.
+    def not_loaded
```

Comment:
:+1: mutators [should end in bangs](http://stackoverflow.com/questions/612189/why-are-exclamation-marks-used-in-ruby-methods).

> On Mar 10, 2016, at 2:21 PM, Thomas Johnson <notifications@github.com> wrote:
> 
> In lib/active_triples/persistence_strategies/parent_strategy.rb <https://github.com/ActiveTriples/ActiveTriples/pull/199#discussion_r55734333>:
> 
> > @@ -30,7 +30,21 @@ def persisted?
> >      end
> >  
> >      ##
> > -    # Destroys the resource by removing it graph and references from the 
> > +    # Mark that this resource has not yet been loaded.
> > +    def not_loaded
> Let's add a ! to the method sig: def not_loaded!
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/ActiveTriples/ActiveTriples/pull/199/files#r55734333>.
>